tests/client/components/App.test.js

```diff
@@ -35,3 +36,15 @@ test('<App /> renders WaitIndicator when info.pending is true', () => {
 
   expect(actual).toBe(true)
 })
+
+test('<App/> renders error message when error prop is true', () => {
```

Comment:
A bit pedantic, but please leave a space after the component name, as in the [style guide](https://github.com/airbnb/javascript/tree/master/react#spacing)